### PR TITLE
External stories shortcode

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Adds the `ucf-external-stories` shortcode. Currently, the only parameters are `limit` and `offset` as the external stories on today don't have any tags or categories that could be used to filter the results down.

**How Has This Been Tested?**
The shortcode can be seen in action in dev.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: includes/external-stories-shortcode.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+						'desc'      => 'The title to display before the news feed',
+						'type'      => 'text',
+						'default'   => 'In the News'
+					),
+					array(
+						'name'      => 'Layout',
+						'param'     => 'layout',
+						'desc'      => 'The layout to display the news items',
+						'type'      => 'select',
+						'options'   => UCF_News_Config::get_layouts(),
```
Comment (cjg89) [inline includes/external-stories-shortcode.php:27]: `UCF_News_Config::get_layouts()` isn't currently designed to return layouts for this shortcode.  Update this method or use a different one for returning the available layouts.

https://github.com/UCF/UCF-News-Plugin/blob/master/includes/ucf-news-config.php#L40-L50

File: includes/ucf-news-feed.php [hunk 2]
```diff
@@ -85,6 +85,26 @@ public static function get_news_items( $args ) {
...
+
+			if ( isset( $args['offset'] ) ) {
+				$params['offset'] = $args['offset'];
+			}
+
+			$param_string = urldecode( http_build_query( $params ) );
+
+			$feed_url = $args['feed_url'];
+
+			$url = "$feed_url?$param_string";
```
Comment (cjg89) [inline includes/ucf-news-feed.php:103]: This URL will be invalid if the feed url is modified in the plugin settings field to include extra query params.  Consider at least adding a note to the field description stating that query params aren't supported, or add logic here that separates existing query params in `$feed_url` and merges them with those in `$params`.

File: includes/external-stories-shortcode.php (hunk 1 above)
Comment (jmbarne3) [reply includes/external-stories-shortcode.php]: Created a new function for pulling layouts.

File: includes/ucf-news-feed.php (hunk 2 above)
Comment (jmbarne3) [reply includes/ucf-news-feed.php:103]: Updated the message for this setting for the time being. We'll make a utility function in the future to handle params present in the URL.

**Pull request merged**